### PR TITLE
Containerize public-annotations-api

### PR DESCRIPTION
**Pull request by BerniVarga**
I will complete the PR with the version number in the service file, as soon as my changes get merged in master. 
Otherwise, it's ready and tested in XP.

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -130,6 +130,10 @@ services:
 - name: notifications-rw@.service
   version: v30
   count: 2
+- name: public-annotations-api-sidekick@.service
+  count: 2
+- name: public-annotations-api@.service
```
Comment (catalin-ilea) [inline services.yaml]: Please enable zero downtime deployment

File: public-annotations-api-sidekick@.service [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+After=public-annotations-api@%i.service
+
+[Service]
+ExecStart=/bin/sh -c "\
+  etcdctl set   /ft/services/public-annotations-api/healthcheck     true; \
+  etcdctl mkdir /ft/services/public-annotations-api/servers; \
+  etcdctl set /ft/healthcheck/public-annotations-api-%i/path /__health; \
+  while true; do \
+    PORT=`[ $PORT ] && echo $PORT || echo $(/usr/bin/docker port public-annotations-api-%i 8080 | cut -d':' -f2)`; \
+    etcdctl set /ft/services/public-annotations-api/servers/%i "http://$HOSTNAME:$PORT" --ttl 600; \
```
Comment (catalin-ilea) [inline public-annotations-api-sidekick@.service]: Can you please use the pattern below? 
https://github.com/Financial-Times/up-service-files/blob/master/document-store-api-sidekick%40.service#L17

File: public-annotations-api@.service [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+KillMode=none
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p-%i > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p-%i > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c '/usr/bin/docker history coco/public-annotations-api:$DOCKER_APP_VERSION > /dev/null 2>&1 || /usr/bin/docker pull coco/public-annotations-api:$DOCKER_APP_VERSION'
+
+ExecStart=/bin/sh -c '\
+  export APP_PORT=8080; \
+  export ENV=$(/usr/bin/etcdctl get /ft/config/environment_tag); \
+  export NEO_HOST_PORT=$(/usr/bin/etcdctl get /ft/services/neo4j/servers/1); \
+  /usr/bin/docker run --rm --name %p-%i -p $APP_PORT --env "NEO_URL=$NEO_HOST_PORT/db/data" --env "APP_PORT=$APP_PORT" --env "GRAPHITE_ADDRESS=graphite.ft.com:2003" --env "GRAPHITE_PREFIX=coco.services.$ENV.public-annotations-api.%i" --env "LOG_METRICS=true" --env "CACHE_DURATION=1h" coco/public-annotations-api:$DOCKER_APP_VERSION;'
```
Comment (catalin-ilea) [inline public-annotations-api@.service]: Can we please have "LOG_METRICS=false"? Having the metrics is graphite is enough

Comment (catalin-ilea) [inline public-annotations-api@.service]: "CACHE_DURATION=1h" What are we caching here?

File: services.yaml (hunk 1 above)
Comment (BerniVarga) [inline services.yaml]: Done.

File: public-annotations-api-sidekick@.service (hunk 2 above)
Comment (BerniVarga) [inline public-annotations-api-sidekick@.service]: ok

File: public-annotations-api@.service (hunk 3 above)
Comment (BerniVarga) [inline public-annotations-api@.service]: LOG_METRICS is set to false.

**Comment by tosan88** [comment]
:+1:

**Comment by catalin-ilea** [comment]
lgtm

**Pull request merged**